aiida/backends/sqlalchemy/manager.py

```diff
@@ -62,7 +62,7 @@ def reset_backend_environment(self):
         from aiida.backends import sqlalchemy
         if sqlalchemy.ENGINE is not None:
             sqlalchemy.ENGINE.dispose()
-        sqlalchemy.SCOPED_SESSION_CLASS = None
+        sqlalchemy.SESSION_FACTORY = None
```

Comment:
Shouldn't this method simply call `aiida.backends.sqlalchemy.reset_session()`?

Reply:
Sure does :)